gammapy/datasets/spectrum.py

```diff
@@ -607,7 +616,7 @@ def peek(self, figsize=(16, 4)):
         ax1.set_title("Counts")
 
         if isinstance(self, SpectrumDatasetOnOff) and self.counts_off is not None:
-            self.background.plot_hist(ax=ax1, label="alpha * N_off")
+            self.background.plot_hist(ax=ax1, label="background")
```

Comment:
I guess that `peek` might return `alpha * n_off`. In particular, since no source model is actually displayed.

But I have no strong opinion there. Of course one needs to document properly what is actually plotted.